File: server/handler/request/tag.go
package request

// CreateTagRequest 创建标签请求
type CreateTagRequest struct {
	Name        string `json:"name" binding:"required,min=1,max=50"`
}

// UpdateTagRequest 更新标签请求
type UpdateTagRequest struct {
	Name        string `json:"name" binding:"required,min=1,max=50"`
}

// CreateTagsRequest 批量创建标签请求
type CreateTagsRequest struct {
	Names []string `json:"names" binding:"required,dive,min=1,max=50"`
}

File: admin/src/types/tag.ts
// 标签实体
export interface Tag {
  id: number
  name: string
  created_at: string
  updated_at: string
}

// 创建标签请求
export interface CreateTagRequest {
  name: string
}

// 更新标签请求
export interface UpdateTagRequest {
  name: string
}

// 批量创建标签请求
export interface CreateTagsRequest {
  names: string[]
}
